tools/testing-server/plugins/bam-parser/index.js

```diff
@@ -2,7 +2,7 @@ const fp = require('fastify-plugin')
 const querypack = require('@newrelic/nr-querypack')
 
 const beaconRequestsRegex = new RegExp(
-  '^/1/|/events/1/|/jserrors/1/|/ins/1/|/resources/1/|/blob',
+  '^/1/|/events/1/|/jserrors/1/|/ins/1/|/resources/1/|/browser/blobs',
```

Comment:
does blobs need to end with a `/` for sure measures?

Reply:
RegExp automatically appends the slash at the end, is that what you mean here?  The browser/blob path does not have any other subpaths on it, only QPs and a body.  LMK if thats not what you meant here

<img width="597" alt="Screenshot 2023-07-13 at 10 23 08 AM" src="https://github.com/newrelic/newrelic-browser-agent/assets/24296585/7a942435-072a-40cc-adbd-33fa61449617">

Reply:
Like the other endpoints end with a `/` (or in regex translation `\/`) , e.g. `/1/` | `\/1\/` or `/events/1/` | etc. 
Just wondering if you should check for `\/blobs\/` although I don't expect a problem with a request url path end up being like `/browser/blobnotreally/asdf?...` so you can nvm this.